### ISSUE TITLE
Problems using stylelint with npm 

### ISSUE THREAD

**Issue by friendlymachine** [stylelint/stylelint#599]
I have a build process that using npm and I have been trying to add stylelint without success.

First of all, I wasn't able to use stylelint on the command line until I installed it globally. Shouldn't it be available to the local project without being installed globally?

The other issue I'm seeing is that stylelint throws errors when used in within my npm scripts. For example, this works on the command line provided stylelint is installed globally:

`stylelint 'css/**/*.css'`

However, when I add this task to npm scripts:

`"lintCSS": "stylelint 'css/**/*.css'"`

And then go `npm run lintCSS` I get the following error:

```
SyntaxError: Unexpected token }
    at Object.parse (native)
    at exports.parse (/Users/John/NodeApps/reactor/node_modules/rc/lib/utils.js:15:17)
    at addConfigFile (/Users/John/NodeApps/reactor/node_modules/rc/index.js:31:20)
    at module.exports (/Users/John/NodeApps/reactor/node_modules/rc/index.js:45:3)
    at buildConfig (/Users/John/NodeApps/reactor/node_modules/stylelint/dist/postcssPlugin.js:102:78)
    at /Users/John/NodeApps/reactor/node_modules/stylelint/dist/postcssPlugin.js:46:18
    at LazyResult.run (/Users/John/NodeApps/reactor/node_modules/postcss/lib/lazy-result.js:206:20)
    at /Users/John/NodeApps/reactor/node_modules/postcss/lib/lazy-result.js:120:37
    at LazyResult.asyncTick (/Users/John/NodeApps/reactor/node_modules/postcss/lib/lazy-result.js:134:15)
    at processing.Promise.then._this2.processed (/Users/John/NodeApps/reactor/node_modules/postcss/lib/lazy-result.js:160:20)
```

Any ideas?

**Comment by davidtheclark** [comment]
Which version of stylelint do you have installed?

Also, can you please post your whole package.json?

**Comment by friendlymachine** [comment]
Thank you for your fast reply! After I submitted this I kept trying to debug. It looks like the problem was with the version of stylelint. I had 3.20 specified in `package.json` but 2.3.2 was what was actually installed. Updating with npm wasn't working for some reason. I had to manually delete stylelint from `node_modules` and then reinstall. That seemed to resolve the problem. Again, thank you for your reply. I'm going to close this issue.
